### PR TITLE
feat(FEC-8153): save reference to player instances and expose API to get them

### PR DESCRIPTION
**Pull request by OrenMe**
### Description of the Changes

Save reference to each instantiated player.
Add methods to get all players or a player by ID.
Make sure to delete the reference upon destroying a player.

- will add tests once review is done.
- still deliberating if to add this code in its own module. doing so will add some boilerplate code of module management which i'm not sure is worth the addition, i.e. need to add register player and delete player methods....

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/kaltura-player.js [hunk 1]
```diff
@@ -28,6 +28,28 @@ import {
   Utils
 } from '@playkit-js/playkit-js';
 
+type playersType = {[id: string]: Object};
```
Comment (dan-ziv) [inline src/kaltura-player.js]: I don't see how this type is necessary...
But if so, it should be:
`type PlayerInstances = {[id: string]: KalturaPlayer};`

Comment (dan-ziv) [reply src/kaltura-player.js]: If you still use this type, move it to `/flow-typed` and doc it with `@typdef`

File: src/kaltura-player.js [hunk 2]
```diff
@@ -28,6 +28,28 @@ import {
...
+function getPlayers(): playersType {
+  return Players;
+}
+
+/**
+ * get a player instance by id
+ * @param {string} id - the player ID
+ * @returns {Object | null} player - the player if found by the supplied ID or null if key doesn't exist
+ */
+function getPlayer(id: string): ?Object {
```
Comment (dan-ziv) [inline src/kaltura-player.js]: `function getPlayer(id: string): ?KalturaPlayer {`

File: src/kaltura-player.js [hunk 3]
```diff
@@ -41,6 +63,7 @@ class KalturaPlayer extends FakeEventTarget {
     super();
     this._eventManager = new EventManager();
     this._localPlayer = loadPlayer(options);
+    Players[options.targetId] = this;
```
Comment (dan-ziv) [inline src/kaltura-player.js]: Add this sentence as the constructor last operation

**Comment by OrenMe** [comment]
@dan-ziv please review CR changes

**Pull request merged**